DataSetExplorer/ConsoleApp/ConsoleAppMainMenu.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Specialized;
+using FluentResults;
+
+namespace DataSetExplorer.ConsoleApp
+{
+    class ConsoleAppMainMenu
```

Comment:
Violation of DRY: MainMenu would suffice, as it is part of the ConsoleApp namespace.